generator/generator_tests/metadata_parser_test.cpp

```diff
@@ -270,3 +270,196 @@ UNIT_CLASS_TEST(TestWithClassificator, Metadata_ValidateAndFormat_duration)
   test("P4D", "");
   test("PT50:20", "");
 }
+
+
+UNIT_CLASS_TEST(TestWithClassificator, ValidateAndFormat_facebook)
+{
+  FeatureBuilderParams params;
+  MetadataTagProcessor p(params);
+  Metadata & md = params.GetMetadata();
+
+  p("contact:facebook", "");
+  TEST(md.Empty(), ());
+
+  p("contact:facebook", "osm.us");
+  TEST_EQUAL(md.Get(Metadata::FMD_FACEBOOK_PAGE), "osm.us", ());
+  md.Drop(Metadata::FMD_OPERATOR);
```

Comment:
Why are you always dropping operator? How is it related to social pages?

Reply:
Copy-n-paste issue. Fixed unit-tests